Keyboard/DivvunSpellBannerPlugin.swift

```diff
@@ -46,13 +48,7 @@ extension DivvunSpellBannerPlugin: BannerViewDelegate {
     }
 
     public func didSelectBannerItem(_ banner: BannerView, item: BannerItem) {
-        if let value = item.value as? String, value == "error" {
-            banner.items = []
-            banner.isHidden = true
-            return
-        }
```

Comment:
Doing a global search, I couldn't find anywhere in the project where a `BannerItem` is created with a `value` of `"error"`. So pretty sure this is dead code. Perhaps I'm missing something.